sensorlib/src/sensor/SemanticLidarSensor.py

```diff
@@ -519,3 +522,38 @@ def update_object_frame_and_timestamps_from_hitpoint(self, obj, timestamp):
                        timestamp=timestamp,
                        position=new_position
                        )
+
+    def update_velocity_angularVelocity(self, detected_objects):
+        for detected_object in detected_objects:
+            print(detected_object.rotation)
+            if detected_object.objectId in prev_objects:
+                time_diff = detected_object.timestamp - prev_objects[detected_object.objectId]['timestamp']
+                if  time_diff and np.all(detected_object.velocity==0) and not (prev_objects[detected_object.objectId]['pose'] == detected_object.position).all():
+                    pose_diff_x = ((detected_object.position[0] - prev_objects[detected_object.objectId]['pose'][0])**2)**0.5
+                    pose_diff_y = ((detected_object.position[1] - prev_objects[detected_object.objectId]['pose'][1])**2)**0.5
+                    pose_diff_z = ((detected_object.position[2] - prev_objects[detected_object.objectId]['pose'][2])**2)**0.5
```

Comment:
This is effectively making the pose_diff positive all the time. We don't need the `**2)**0.5` part